src/components/Calculator.js

```diff
@@ -0,0 +1,58 @@
+import { useState } from 'react';
+import calculate from '../logic/calculate';
+
+const obj = {
+  total: null,
+  next: null,
+  operation: null,
+};
+
+export default function Calculator() {
+  const [output, setOutput] = useState(obj);
+
+  const gumbs = ['AC', '+/-', '%', '÷', 7, 8, 9, 'x', 4, 5, 6, '-', 1, 2, 3, '+', 0, '.', '='];
+  const handleBtnClick = (e) => {
+    const btn = e.target.name;
+    return setOutput(calculate(output, btn));
+  };
+
+  const screen = (item) => {
+    if ((!item.total) && (!item.next)) {
+      return '0';
+    }
+    if (!item.total) {
+      return item.next;
+    }
+    if (item.next && item.total && item.operation) {
+      return item.next;
+    }
+    return item.total;
+  };
+
+  return (
+    <section id="calculator">
+      <input
+        type="text"
+        value={screen(output)}
+        disabled
+        id="exit"
+      />
+      <section id="input">
+        <article id="digits">
+          {
+            gumbs.map((gumb) => (
+              <button
+                key={gumb}
+                type="button"
+                name={gumb}
+                onClick={handleBtnClick}
+              >
+                {gumb}
+              </button>
```

Comment:
- [ ] Great job on creating a functional app! However, one area for improvement is the use of props. I would recommend creating a button component that accepts props(eg: type,name, etc) form its parents. This will not only make your app more organized, but also result in cleaner code. Keep up the good work!
![image](https://user-images.githubusercontent.com/43877689/221365232-e7bfa5a0-3393-4c8e-b53f-2a510bc47aea.png)